run.py

```diff
@@ -41,9 +52,14 @@ def create_app(config_filename: str) -> Flask:
 @application.before_first_request
 def create_tables():
     from app.database.sqlalchemy_extension import db
-
     db.create_all()
 
 
 if __name__ == "__main__":
     application.run(port=5000)
+
```

Comment:
can you add a comment here, to explain you are adding cli options or something, also should this be inside or outside the create_app()?